pkg/scheduler/internal/queue/scheduling_queue_test.go

```diff
@@ -376,11 +376,17 @@ func BenchmarkMoveAllToActiveOrBackoffQueue(b *testing.B) {
 	}
 
 	podTemplates := []v1.Pod{highPriorityPod, highPriNominatedPod, medPriorityPod, unschedulablePod}
+	plugins := []string{"foo", "bar", "baz"}
+	events := []string{NodeAdd, PodAdd}
 
 	for _, tt := range tests {
 		b.Run(tt.name, func(b *testing.B) {
 			c := clock.NewFakeClock(time.Now())
-			q := NewPriorityQueue(newDefaultQueueSort(), WithClock(c))
+			m := map[framework.ClusterEvent]sets.String{
+				{Resource: framework.Node, ActionType: framework.Add}: sets.NewString("foo"),
```

Comment:
Make this as realistic as possible. Can we use all the events we know of?

Reply:
> Can we use all the events we know of?

The events (NodeAdd, PodAdd) are real ones. You meant real plugin names to replace foo/bar here?

Reply:
What I want to see is a realistic size of the maps that we will have in production.

Reply:
Per discussion in https://github.com/kubernetes/kubernetes/pull/99260#discussion_r581141543, I will go compose this unit benchmark test covering podMatchesEvent only, and use integrate test to benchmark the entire e2e flow. Please see latest commit.